apps/teamsjs-cc-app/tabs/src/components/capabilities/pages/ReturnFocusToAppBar.tsx

```diff
@@ -0,0 +1,38 @@
+import * as Fluent from "@fluentui/react-northstar";
+
+import { app, pages } from "@microsoft/teams-js";
+
+/**
+ * This component returns a button which returns focus to app bar.
+ */
+export const ReturnFocusToAppBar = () => {
+  // cCheck if app is initialized;
+  if (app.isInitialized()) {
+    return (
+      <>
+        {pages.isSupported() && (
+          <Fluent.Segment className="ui-pagessegment">
+            <Fluent.Header content="Return Focus To App Bar" as="h2" />
+            <Fluent.Flex gap="gap.small" vAlign="center">
+              <Fluent.Text
+                className="ui-pagestext"
+                content="Returns focus to the host's App bar. (Curently works only in teams)"
```

Comment:
Curious: are other hosts saying that all of pages is supported but not supporting this function? If so, is that something we need to follow up with them on?

Reply:
Great call out I have updated this in the internal cap matrix to get the hosts to update